.github/workflows/ci.yml

```diff
@@ -45,7 +45,7 @@ jobs:
           creds: ${{ secrets.AZURE_ACI_CREDENTIALS }}
       - name: Setup RavenDB
         id: setup-ravendb
-        uses: Particular/setup-ravendb-action@v1.8.0
+        uses: Particular/setup-ravendb-action@f170fb683daffff53634789183786ab5731bbb28
```

Comment:
This needs to be changed to point to the next tag once https://github.com/Particular/setup-ravendb-action/pull/64 is merged